scripts/generators/safe_struct_generator.py

```diff
@@ -41,8 +41,6 @@ def __init__(self):
             'VkMicromapBuildInfoEXT',
             'VkAccelerationStructureTrianglesOpacityMicromapEXT',
             'VkAccelerationStructureTrianglesDisplacementMicromapNV',
-            # The VkDescriptorType field needs to handle every type which is something best done manually
-            'VkDescriptorDataEXT',
```

Comment:
I originally listed them here, doubled checked now, we don't do this manually and instead can auto generate it with `self.union_of_pointers = [ ]`